### DISCUSSION TITLE
Netlify proxy randomly returns 500

### DISCUSSION THREAD

**Discussion by kevlatus** [plausible/analytics#1157] [Anything Else]
Recently I switched to the Netlify redirect proxy described [in this guide](https://plausible.io/docs/proxy/guides/netlify). Unfortunately, the redirects return code 500 when hitting the page for the first time in a while (at least that’s the pattern, which I observed). After waiting for a couple of minutes the redirect works perfectly fine.

Does someone know what's happening here or has encountered similar issues?

**Comment by ukutaht** [comment]
This came u but I thought we had solved it: https://github.com/plausible/analytics/issues/1085

Does your error look similar?
